packages/shared/src/graphql/feed.ts

```diff
@@ -178,7 +178,7 @@ export const SOURCE_FEED_QUERY = gql`
       ...FeedPostConnection
     }
   }
-  ${getFeedPostFragment('pinnedAt')}
+  ${getFeedPostFragment('pinnedAt content')}
```

Comment:
Not specific to this PR but I was thinking by returning `content` for feed listing which has `4000` char limit we might have considerably bigger responses if there is more freeform posts in the future. @idoshamun FYI interested what you think.

Reply:
That's a good point! I also think that we don't need for post modal in 99% of the cases. We will need it only for edit mode. Let's log the post modal as something to fix and in this PR let's remove the content for the feed

Reply:
For the feed, we will need the content (switch to using `contentHtml` now) on the feed to display when there is no cover image on the post. Do we maybe want to make it conditional as to only return it when there is no image?